roles-in-tech/security.md

```diff
@@ -1,71 +1,114 @@
 
-### Independent Practice
+- **If wi-fi has a passwords, then it is secure** - If the person who travels to different workplaces and needs to use public wi-fi it is not recommended to do sensitive transactions through it. The same password is shared with many users and anyone can see the information that is being transferred.
 
-Build a page template to make it easier to prevent CSRF.
+- **By turning off GPS, you will not be traced** - Almost half of people surveyed either didn't know or were unsure whether disabling GPS prevents all tracking. Your phone's built-in location service is just one way data can be collected. As Pew Research notes, your phone also connects to cell towers and Wi-Fi networks, and connection to these technologies allows tracking too.
 
-### Challenge
+- **Emails are secure** - Over 400 businesses are targeted daily, and these emails are growing increasingly and it is challenging to identify since they’re designed to mimic legitimate communication so they can gain access to your sensitive information – including usernames and passwords.
 
-If you are feeling inclined, or interested in red teaming, you can experiment with [Insecure Labs](http://www.insecurelabs.org) or a [Kali Linux](https://www.kali.org/) VM and read about its rich [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) tool suite.
+- **Small organizations will not be attacked** - Major organizations assume that they are small and will not be targeted as we are small company. The research shows that 70% of cyber attacks target small businesses. So today for everyone it is very important to take care of its security if its data is worth value for money, whether it is a website for fun or it is a huge organization.
+
+### Guided Practice
+- *Talk to as many professionals as you can*
+
+ [National Cyber Security Centre](https://live.ncsc.gov.uk/topics/new-talent) is worth for who lives in UK
+
+### Independent Practice
+- *You can play a game* called [PicoCTF](https://picoctf.com/about) that involves using reverse engineering, hacking, decryption and other skills to resolve the mystery behind the sudden disappearance of your character’s father.
+
+- *Experiment with second computer*. Firstly, install Linux, BSD, Qubes OS and other operating systems on it.
+Then, plug it to an Arduino board (i.e. an electronics platform that can read input like a light or touch sensor or your social media messages), and turn it into an output (publishing something on Twitter, switching on an LED or whatever).
+Don’t forget. Install security and (with permission) monitor the networks that you connect to.
 
 ### Check for Understanding
 
-The next step is to try to XSS each other's sites.
+- [27 ways to learn how to code on the cheap (or free)](https://thenextweb.com/dd/2017/04/03/so-you-want-to-be-a-programmer-huh-heres-25-ways-to-learn-online/#.tnw_hZDm58Mu)
 
 ## Understanding the Field
```

Comment:
Move "Understanding the field" and "What kinds of security jobs" sections to before "guided practice".